CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ that users understand how the changes affect the new version.
 -->
 version 5.0.1
 ---------------------------
++ Smoove: enable genotyping
++ Bcftools: add boolean option to remove uncalled genotypes.
```

Comment:
Is this in vcftools, there don't seem to be any actual changes there (just removed whitespaces)?

Reply:
Oops sorry about that, I forgot I already had that change.